pkg/requests/base.go

```diff
@@ -297,6 +321,40 @@ func (rb *Base) buildDataForRequest(params *RequestParameters) (string, error) {
 	return encode(keys, values), nil
 }
 
+func (rb *Base) buildMultiPartRequest(params *RequestParameters) (*bytes.Buffer, string, error) {
```

Comment:
We need some comments to clarify how to use this method.
```suggestion
// buildMultiPartRequest builds a request that can contain mix of files and plain strings.
// Similar to making a multipart request using curl, add the local filepath to params arg with @ prefix.
// e.g. params.AppendData([]string{"photo=@/path/to/local/file.png"})
func (rb *Base) buildMultiPartRequest(params *RequestParameters) (*bytes.Buffer, string, error) {
```